cfgov/login/urls.py

```diff
@@ -33,6 +33,12 @@
 else:
     urlpatterns = [
         re_path(r"^login/$", LoginView.as_view(), name="cfpb_login"),
+        # Override Wagtail's login URL.
+        # By overriding the path and name here, we avoid heavily customizing
+        # the wagtailadmin/login.html template.
+        re_path(
+            r"^admin/login/$", LoginView.as_view(), name="wagtailadmin_login"
+        ),
```

Comment:
I tested this and we don't actually need these lines, as they duplicate the URL pattern that's already defined in Wagtail's admin [urls.py](https://github.com/wagtail/wagtail/blob/2b3da3582eeb53d4fe543fcbae91c3f663b7f8f5/wagtail/admin/urls/__init__.py#L121). The template override gets picked up whether we define a URL override or not.